app/services/middleware_topology_service.rb

```diff
@@ -11,11 +11,11 @@ class MiddlewareTopologyService < TopologyService
       :middleware_deployments,
       :middleware_datasources,
       :middleware_messagings,
-      :lives_on => [:host]
+      :lives_on => [:host, :container]
```

Comment:
it works, the linking is done [here](https://github.com/ManageIQ/manageiq-providers-hawkular/blob/master/app/models/manageiq/providers/hawkular/inventory/parser/middleware_manager.rb#L43) and the `lives_on` is currently overloaded for two purposes. And it depends on the https://github.com/ManageIQ/manageiq-ui-classic/pull/1221 so that it will not fail in case there is `:lives_on :host` situation and trying to call `:container` and vice-versa.